spec/lib/rmt/scc_spec.rb

```diff
@@ -25,7 +25,9 @@
         [product, extension].each do |product|
           db_product = Product.find(product[:id])
           db_product.attributes.each do |key, value|
-            expect(value).to eq(product[key.to_sym])
+            if product[:id] != 900000
```

Comment:
Why do we skip this product?

Reply:
this was still a relic from the first implementation, that removed the "beta" at import, so then the test would fail because the value in the database would be different to the "sent" data.
I removed this now, as it is not needed anymore